src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -234,7 +236,7 @@ public bool DisableParallelization
     }
 
     /// <summary>
-    /// Gets or sets the Target platform this run is targeting. Possible values are <c>x86|x64|arm|anycpu</c>.
+    /// Gets or sets the Target platform this run is targeting. Possible values are <see cref="Architecture"/> except for AnyCPU or Default.
```

Comment:
Not sure but I would have said `and` instead of `or`.

```suggestion
    /// Gets or sets the Target platform this run is targeting. Possible values are <see cref="Architecture"/> except for AnyCPU and Default.
```

Reply:
Seems equivalent, but I changed it anyway.